README.md

```diff
@@ -1,8 +1,14 @@
+# Jazz History Database Content Management System
+
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 You can clone the project by running
 ### `git clone https://github.com/jazzhistorydatabase/jhdb-cms.git`
 
+*NOTE: Running this app requires unchecked credentials files for the server and client. (Development access to Google Cloud is required in order to download using the links below)*
+- [Server credentials](https://storage.cloud.google.com/jhdb-cms_cloudbuild/server-creds-dev.json): rename to `server-creds.json` and place in project root 
```

Comment:
had to rename to server-creds-staging.json; server.creds on its own was not recognized, not sure if something else needs to be changed there